### ISSUE TITLE
[Discussion] Future of Policeman?

### ISSUE THREAD

**Issue by 5t3f4n** [futpib/policeman#197]
This extension looked very promising when I first found it, but I can't help but notice that development seems slow lately. There are many issues/requests that remain unanswered, let alone fixed. There hasn't been a single new release (and only a couple pre) since I started using it early this year. Not a whole lot of commits either.

Recently, [uMatrix](https://github.com/gorhill/uMatrix) has been ported to Firefox and is now a direct "competitor" to Policeman. I imagine this is going to become quite popular compared to Policeman due to uMatrix already having a user base on other platforms as well as the [developer](https://github.com/gorhill) having a bit of a "fan base" due to his development of [uBlock](https://github.com/gorhill/uBlock). I've also seen requests for Policeman to mimic the UI of uMatrix in the past, so I expect many users will jump ship based on this.

Maybe instead of beating a dead horse, Policeman developers could help improve uMatrix instead as it's still in its infancy as far as Firefox support goes. uMatrix was originally developed for Chromium-based browsers so they could probably use your expertise with Firefox. It would be a shame to see all the work put into Policeman just wither away.

I'd like to hear developers' thoughts as well as user input on this.

**Comment by Firminator** [comment]
IMHO uMatrix GUI (for Firefox) seems rather convoluted, unintuitive and generally Beta'esque compared to Policeman GUI.

**Comment by Radagast** [comment]
I personally think the UI for both policeman and umatrix leaves quite a lot to be desired. However, uMatrix and for that matter, Request Policy are both under active development and both, as far as I know, support e10s, whereas Policeman development seems to have stalled and as far as I can tell, with e10s enabled, things break.

It would be nice to know if Policeman has a future...

**Comment by 5t3f4n** [comment]
Yes, a lot of things can be said about the UI of Policeman/uMatrix/HTTPSwitchboard and the likes, but I believe that's just something we have to accept for a power-user extension like this. They are no lookers, but workflow is the most important part here. uMatrix will almost always get the job done in fewer steps while giving a better overview, thus saving both clicks and time.

But my main concern is that Policeman seems to have stalled in development, all while alternatives are already prepared for the changes to come in Firefox. Even if someone decides to pick up the ball again, it may be too late to start playing catch-up at this point.
